tests/grid/grid_generator_replicate_tria_01.cc

```diff
@@ -0,0 +1,70 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 - 2017 by the deal.II authors
```

Comment:
Just use "Copyright (C) 2018 by the deal.II authors".

Reply:
Do I still change it to "Copyright (C) 2018 by the deal.II authors" now, or change it to 2019

Reply:
Just put `2018 - 2019`.

Reply:
A silly question: how to generate the output file? I haven't done this for a while, so I followed instructions here: https://www.dealii.org/current/developers/testsuite.html#layoutaddtests.

As I need to update the output file, I replaced the existing one `grid_generator_replicate_triangulation_01.output` with an empty file and in the dealii/tests I did:
```
make setup_tests
ctest -V -R grid/grid_generator_replicate_triangulation_01
``` 
It failed at the comparison stage. I went to grid/grid_generator_replicate_triangulation_01.release (and .debug), yet I didn't find any file with the name "output". Do I misunderstand anything?

Thanks

Reply:
Hmm... There should be a file named `output` in `${BUILD_DIRECTORY}/tests/grid/grid_generator_replicate_triangulation_01.debug`.

Reply:
My system is OSX mojave. Not sure if this affects testing.

If I do the two steps mentioned above in the directory where I build dealii (so it is the temporary folder that I will delete later), I got the following:
```
UpdateCTestConfiguration  from :/Users/GrillCheese/contribution/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/Users/GrillCheese/contribution/build/DartConfiguration.tcl
Test project /Users/GrillCheese/contribution/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
No tests were found!!!
```

If I go to the directory where my local repo is stored and do the same thing, test runs and fails but I cannot find the output.

Is this how it's supposed to be done? What does ${BUILD_DIR} mean?

Reply:
As long as the file `grid_generator_replicate_triangulation_01.output` exists, `ctest -V -R grid/grid_generator_replicate_triangulation_01` should find the test. Maybe, you deleted the file accidentally or didn't run `make setup_tests`?

Reply:
In other words, create a dummy (empty) output file for the test, run `make setup_tests` again, run the test (it will fail because its output is different from the -- empty! -- stored output file), and then copy the correct output file over the dummy file you had created earlier.